### DISCUSSION TITLE
Disable internal Grafana alert

### DISCUSSION THREAD

**Discussion by DanielCastronovo** [grafana/grafana#59872] [Alerting]
Hello,

I would like to disable Grafana internal alerting, but not disable the alerting center because we used the alertmanager component.

**Comment by VermaPriyanka** [comment]
Grafana users who plan to use Grafana Alerting only for managing alerts in external systems like Mimir, Loki, Cortex or Prometheus, would benefit from this feature. It can be implemented as an ini setting, perhaps, `grafana_managed_alerts`, which would allow disabling/graying out the UI elements related to Grafana Managed alerts from the Grafana alerting interface.

**Comment by sakjur** [comment]
I believe that #64941 should fix this in a future Grafana release.
- https://github.com/grafana/grafana/pull/64941

**Reply by VermaPriyanka** [reply]
I think that applies if alerting is disabled, but the above use-case requires Grafana alerting to be enabled to allow alert management for external datasources(cortex, mimir, loki, prometheus).

The scenario discussed above is when customers do not want to use/create Grafana managed alerts and do not want to show those UI elements to avoid end-user confusion.

> it's possible (although very unlikely) customers have disabled both legacy and unified alerting, but are still using one of Machine learning, OnCall or Incident

**Comment by DanielCastronovo** [comment]
Thanks @VermaPriyanka
This request been prioritised?

**Comment by DanielCastronovo** [comment]
Any news ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
